adapter/config/types.go

```diff
@@ -189,6 +190,14 @@ type globalCors struct {
 	ExposeHeaders    []string
 }
 
+// Envoy request body manipulating configurations
+type payloadPassing struct {
```

Comment:
We can update the comment. Also the same comment as https://github.com/wso2/product-microgateway/pull/2653/files#r802290271